Java Migration/TFS_2017 RTW.md

```diff
@@ -0,0 +1,57 @@
+Below diagram explains the steps to migrate from Oracle JRE to Azul Zulu OpenJDK for TFS 2017 RTW.
+
+![Java Migration flow](https://github.com/msftazdev/Code-Search/blob/msftazdev-patch-1/Java%20Migration/flow2.png)
+
+## Step 1: Pause Search indexing
+Go to https://github.com/Microsoft/Code-Search and find the right folder based on the TFS version you are using. For TFS 2017 RTW, go to https://github.com/Microsoft/Code-Search/tree/master/TFS_2017RTW. To pause all indexing, execute the script PauseSearchIndexing.ps1 on TFS server machine from Windows PowerShell with administrative privileges. You will be prompted to enter:
+
+* The SQL server instance name where the TFS configuration database resides.
+* The name of the TFS configuration database.
+
+## Step 2: Stop Elasticsearch Service
+Open Command Prompt as an administrator 
+
+If Elasticsearch is installed on the same server as TFS (local installation), use below command to locate ES folder. For remote search installations, locate the ES installation path and change the directory accordingly.
+### Change directory: 
+cd "C:\Program Files\Microsoft Team Foundation Server 15.0\Search\ES\elasticsearch-1.7.1-SNAPSHOT\bin"
+### Stop the service:
+execute "service.bat stop"
+
+## Step 3: Remove Elasticsearch Service
```

Comment:
Is there a reason why it needs to be removed for this version instead of starting/stoping like in the others?

Reply:
For TFS 2017 RTW, ES behavior is different (because it is a different version which is 1.7x unlike others which are 2.x+). That's the reason 2 additional steps of remove and install only for TFS 2017 RTW